docs/addons/kafka.md

```diff
@@ -43,8 +43,7 @@ Currently, use the `nav-dev` pool for development, and `nav-prod` for production
 | `nav-dev` | 2 | 3 | `dev-gcp` | `dev-gcp`, `dev-fss`, `dev-sbs` |
 | `nav-prod` | 2 | 3 | `prod-gcp` | `prod-gcp`, `prod-fss`, `prod-sbs` |
 
-{% tabs %}
-{% tab title="topic.yaml" %}
+
 ```yaml
```

Comment:
Ser ut som du bare har fjernet tabs her. 

```
=== ".github/workflows/deploy.yml"
    ```yaml
    name: Build, push, and deploy
```